components/ServiceSelection.vue

```diff
@@ -49,7 +49,7 @@
 </template>
 
 <script>
-import { DModal } from '@brown-ccv/disco-vue-components';
+import DModal from '@/components/disco-vue-components/d-modal';
```

Comment:
Should we follow the Vuex Style guide for naming the components?
https://vuejs.org/v2/style-guide/#Base-component-names-strongly-recommended

If we follow the current structure in the repo, we would have the modal be at `@components/base/Modal`

I don't think we need to keep the `d` or `disco` references. Although I guess we could do the transition in stages...

Reply:
I agree, but in some ways this already is following that convention of an `app specific prefix`. I think the folder could definitely be renamed to `base` or similar, `disco-vue-components` is long.